db/migrate/20240705082709_create_mail_delivery_failures.rb

```diff
@@ -0,0 +1,11 @@
+class CreateMailDeliveryFailures < ActiveRecord::Migration[7.1]
+  def change
+    create_table :mail_delivery_failures do |t|
+      t.string :email_address
```

Comment:
I think we should add an index on the `email_address` column to so that when we're searching for these records we get results quickly.

https://apidock.com/rails/ActiveRecord/ConnectionAdapters/Table/index

https://stackoverflow.com/questions/1108/how-does-database-indexing-work